lms/static/styles/components/_SubmitGradeForm.scss

```diff
@@ -91,11 +105,19 @@
   top: 0;
 }
 
-.SubmitGradeForm__spinner {
-  $spinner-size: 100px;
-  position: absolute;
-  width: $spinner-size;
-  height: $spinner-size;
-  left: calc(50% - #{$spinner-size});
-  top: calc(50% - #{$spinner-size});
+.SubmitGradeForm__submit-spinner {
+  @include mixins.spinner(150px);
```

Comment:
I'll revert this back to 100px, but there is also a ticket to refactor all the spinner in LMS into one.